Libplanet/Blockchain/BlockChain.cs

```diff
@@ -102,7 +117,8 @@ Block<T> genesisBlock
             IStore store,
             Guid id,
             Block<T> genesisBlock,
-            bool inFork
+            bool inFork,
+            bool render = true
```

Comment:
Since it's just a `private` method, making all parameters required might be better IMHO.

Reply:
I made `render` as a required parameter.